app/controllers/api_controller/authentication.rb

```diff
@@ -23,6 +21,16 @@ def destroy_auth
       render_normal_destroy
     end
 
+    # Used from outside the API, needs to raise errors for invalid userid or requester_type
+    def generate_user_token(userid, requester_type)
+      api_log_info("Generating Authentication Token for user id: #{userid} requester_type: #{requester_type}")
+      User.find_by_userid!(userid)
+      REQUESTER_TTL_CONFIG.fetch(requester_type) if requester_type
```

Comment:
Not sure what the intention is here. Assuming it has some side-effect if the return is not used. Could a comment be added, or perhaps extract this into a method?